en/docs/install.html

```diff
@@ -15,6 +15,7 @@
 {% capture windows      %}{% include_relative _installations/windows.md      %}{% endcapture %}
 {% capture linux        %}{% include_relative _installations/linux.md        %}{% endcapture %}
 {% capture alternatives %}{% include_relative _installations/alternatives.md %}{% endcapture %}
+{% capture nightly      %}{% include_relative _installations/nightly.md      %}{% endcapture %}
```

Comment:
Like a totally separate page? It seemed to fit in nicely here and I imagine this is where people would look for the info, I can put it elsewhere if you like though!